IS7024Project/ParkSchema.jason

```diff
@@ -0,0 +1,674 @@
+﻿{
```

Comment:
created schema for park information for validation use.

Reply:
This appears to be auto-generated.  If that's the case, be aware that it may no longer fit if the data changes.  Enumerations may change, for instance.
Is this used anywhere?  If not, scrap it; it's clutter.